module-cheatsheets/scRNA-seq-advanced-cheatsheet.md

```diff
@@ -167,8 +169,78 @@ Read the full documentation and download cheatsheets (where available) for these
+
+
+### Converting from `SCE` to `Seurat`
+
+The following example code assumes you are starting with an `SCE` object called `sce_object`.
+
+The function `Seurat::as.Seurat()` can be used to convert an `SCE` object into a `Seurat` object and takes the following arguments:
+
+- The `SCE` object to convert
+- Optional named arguments with the following defaults:
+  - `counts = "counts"` specifies that the `SCE` object contains a `"counts"` assay of normalized counts that should be included during conversion.
+    - If there is no `"counts"` assay in the SCE object, set this argument as `counts = NULL` or rename accordingly, e.g. `counts = "whatever_assay_name_you_are_using"`.
+  - `data = "logcounts"` specifies that the `SCE` object contains a `"logcounts"` assay of normalized counts that should be included during conversion.
+    - If there is no `"logcounts"` assay in the SCE object, set this argument as `data = NULL` or rename accordingly, e.g. `data = "whatever_assay_name_you_are_using"`.
+  - `assay = NULL` specifies that, by default, all assays will be converted.
+    - To specify that an additional assay besides `"counts"` or `"logcounts"` should be converted, include it here as in `assay = "additional_assay_name"`.
+  - `project = "SingleCellExperiment"` specifies that the `Seurat` object being created will have this associated project name. You can override this with any string of interest, e.g. `project = "sample_XYZ"`.
+
+
+```r
+# Convert SCE object to Seurat object, assuming both
+#  `counts` and  `logcounts` assays are present
+seurat_object <- Seurat::as.Seurat(sce_object)
+
+# Convert SCE object to Seurat object, where the SCE object
+#  contains a `counts` but not a `logcounts` assay
+seurat_object <- Seurat::as.Seurat(sce_object, data = NULL)
+
+
+```
+
+#### Approaches from `ScPCA`
+
+In addition, this [documentation from the `ScPCA`](https://scpca.readthedocs.io/en/latest/faq.html#what-if-i-want-to-use-seurat-instead-of-bioconductor) introduces how to convert `SCE` objects to `Seurat` objects.
+Although this documentation was written for `ScPCA` datasets, the steps generally apply to any `SCE` object.
+
+We also offer a conversion function `sce_to_seurat()` as part of our [`scpcaTools()` package](https://github.com/AlexsLemonade/scpcaTools/), which holds utilities used in the `ScPCA` workflow.
+Again, although this function was written to convert `SCE` objects from `ScPCA`, it should generally work for most `SCE` objects, although it will only retain a single assay (raw `"counts"`) in the new `SCE` object, and it will not retain reduced dimension representations (e.g., PCA or UMAP).
+Therefore, this function is mostly useful at the early stages of processing before you have normalized counts and and calculated reduced dimensions.
+
+You can obtain this package using the `remotes` package, which may also need to be installed first:
```

Comment:
I don't think we should recommend this at this stage, for a bad reason and a good reason. The bad reason is that I'm just not confident in it yet, and don't want to be debugging it as a general tool at the moment.

The good reason is that we are using R 4.1 on the server, and `scpcaTools` now requires R 4.2, so this won't actually work!

I think it is fine to point people to the code as an example though!

Reply:
Purged `scpcaTools` for R-versioning reasons, and reorganized the remaining text.